apps/website/src/pages/docs/avatar.mdx

```diff
@@ -13,73 +13,53 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<LiveExample src='Avatar.main.tsx'>
-  <AllExamples.AvatarMainExample client:load />
+<LiveExample src='Avatar.initials.tsx'>
```

Comment:
why are we switching the main and initials examples?

Reply:
For the "main" example (shown at the very beginning), let's use a proper avatar with the robot image.

And then later, lets show the three variations with initials, svg and png.

I also think the "Variants" section does not make sense. The example is fine, but I think it should be moved under "Usage" where we explain what `abbreviation` and `image` does. The "Usage" section can also be expanded to include a shortened version of the text from the current "Variants" section.